### DISCUSSION TITLE
How to go back to the repository page when we have searched

### DISCUSSION THREAD

**Discussion by zemse** [community/community#116801] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Currently I am on this screen:
![Screenshot 2024-04-04 at 8 31 20 PM](https://github.com/community/community/assets/22412996/6b59f034-13c0-4e5d-9413-c719e94ae33d)

My goal is to go back to the repository main page, i.e. https://github.com/MaxGraey/as-bignum. I have tried:
- clicking on github logo, it takes me to home page
- clicking on code, it does nothing
- top left hamburger menu, it does not have the link to the main site.
- kept pressing back on my browser, this is annoying if i have performed a lot of searches so i have to traverse back so much.
- i am still ok to just go to address URL bar and then remove bit of stuff (i thought URL might be `https://github.com/MaxGraey/as-bignum/search/div`), but the search URL looks like `https://github.com/search?q=repo%3AMaxGraey%2Fas-bignum+div&type=code`

I remember there was a way, after the github UI update it is very annoying. Please someone help me.

**Comment by brenoSCosta** [comment]
Typically, when I want to return to the main page of a repository, I simply remove the text from the search bar in
 and hit "Enter". For instance, if the current search is repo:MaxGraey/as-bignum div, I just remove "div" from the search and hit "Enter" to return to the repository's main page. Hope this helps!

**Reply by zemse** [reply]
Thanks for sharing this is very helpful

**Comment by andrade** [comment]
I was struggling with this too, and the answer is not obvious at first. Wish they'd fix the UI.

At the top of the page, just below the search bar, there is a button with a cross that says `MaxGraey/as-bignum` (looks like a grayish label or filter button). If you click the button (not the cross!) it takes you back to the repository. In your screenshot, the button is right after "9 files (164 ms) in".

<img width="422" height="83" alt="Screenshot from 2025-10-08 17-43-45" src="https://github.com/user-attachments/assets/a98dd08c-1e15-41df-8336-6018dd584c65" />

For organization-wide searches, however, the button only shows the name of the organization (or user) – `MaxGraey` in this case – so this would not work.
